sklearn/datasets/base.py

```diff
@@ -440,6 +446,10 @@ def load_digits(n_class=10):
         sample, 'target_names', the meaning of the labels, and 'DESCR',
         the full description of the dataset.
 
+    (data, target) : tuple if ``return_X_y`` is True
+
+    .. versionadded:: 0.18
```

Comment:
You're talking about this line ? 
``(data, target) : tuple if return_X_y is True ``